### PR TITLE
John conroy/replace title key with

### PR DESCRIPTION
**Pull request by john-conroy**
Replaces list title as key with a uuid and uses uuid in url route.

### CODE REVIEW

File: context/app/static/js/components/savedLists/AddToList/AddToList.jsx [hunk 1]
```diff
@@ -13,11 +13,12 @@ function AddToList({ selectedLists, addToSelectedLists, removeFromSelectedLists
 
   return (
     <MaxHeightList>
-      {Object.keys(savedLists).map((listName) => (
+      {Object.entries(savedLists).map(([listUuid, value]) => (
```
Comment (mccalluc) [inline context/app/static/js/components/savedLists/AddToList/AddToList.jsx:16]:
```suggestion
      {Object.entries(savedLists).map(([listUUID, value]) => (
```

File: context/app/static/js/components/savedLists/AddToList/AddToList.jsx [hunk 2]
```diff
@@ -13,11 +13,12 @@ function AddToList({ selectedLists, addToSelectedLists, removeFromSelectedLists
...
         <AddToListItem
-          key={listName}
-          title={listName}
-          isSelected={selectedLists.has(listName)}
+          key={listUuid}
+          title={value.title}
+          listUuid={listUuid}
+          isSelected={selectedLists.has(listUuid)}
```
Comment (mccalluc) [inline context/app/static/js/components/savedLists/AddToList/AddToList.jsx:21]:
```suggestion
          key={listUUID}
          title={value.title}
          listUuid={listUUID}
          isSelected={selectedLists.has(listUUID)}
```
(I won't comment on every occurrence below.)

File: context/app/static/js/components/savedLists/listTextFields/index.js [hunk 3]
```diff
@@ -13,7 +13,8 @@ function TitleTextField({ handleChange, title }) {
       label="Title"
       fullWidth
       variant="outlined"
-      placeholder='Like “Spleen-Related Data” or “ATAC-seq Visualizations”'
+      // eslint-disable-next-line prettier/prettier
```
Comment (mccalluc) [inline context/app/static/js/components/savedLists/listTextFields/index.js:16]: Why the change?

**Review by mccalluc — COMMENTED** [review]
- Do a search and replace for `Uuid` -> `UUID`.
- Not sure why the eslint ignore is needed?

Haven't run it locally but looks good: Merge after tweaking. Thanks!

File: context/app/static/js/components/savedLists/listTextFields/index.js (hunk 3 above)
Comment (john-conroy) [reply context/app/static/js/components/savedLists/listTextFields/index.js:16]: This is needed for one of the formatting changes you requested in an earlier PR.
